packages/firestore/src/local/target_data.ts

```diff
@@ -20,59 +20,58 @@ import { Target } from '../core/target';
 import { ListenSequenceNumber, ProtoByteString, TargetId } from '../core/types';
 import { emptyByteString } from '../platform/platform';
 
-/** An enumeration of the different purposes we have for queries. */
+/** An enumeration of the different purposes we have for targets. */
 export enum QueryPurpose {
-  /** A regular, normal query. */
+  /** A regular, normal query target. */
   Listen,
 
   /**
-   * The query was used to refill a query after an existence filter mismatch.
+   * The query target was used to refill a query after an existence filter mismatch.
    */
   ExistenceFilterMismatch,
 
-  /** The query was used to resolve a limbo document. */
+  /** The query target was used to resolve a limbo document. */
   LimboResolution
 }
 
 /**
- * An immutable set of metadata that the local store tracks for each query.
+ * An immutable set of metadata that the local store tracks for each target.
  */
-// TODO(wuandy): rename this to TargetData.
-export class QueryData {
+export class TargetData {
   constructor(
-    /** The query being listened to. */
+    /** The target being listened to. */
     readonly target: Target,
     /**
-     * The target ID to which the query corresponds; Assigned by the
+     * The target ID to which the target corresponds; Assigned by the
      * LocalStore for user listens and by the SyncEngine for limbo watches.
      */
     readonly targetId: TargetId,
-    /** The purpose of the query. */
+    /** The purpose of the target query. */
```

Comment:
Perhaps "target query" => "target" especially if you rename QueryPurpose to TargetPurpose.